### ISSUE TITLE
Add support for satellite image in map

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#119]
## User story
1. As a user
2. I want to display a current satellite imagery map
3. So that I can see how a region looks in real

## Acceptance criteria
* Display current high resolution satalite image
* Overlay POIs (e.g.: charging stations)
* Discuss and document external APIs for fetching the satellite images (Copernicus, OSM, ...)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
so satellite view is more complex than expected. Google maps and similar high quality ones dont want to be included without payment. But for example https://gdz.bkg.bund.de/index.php/default/wms-deutschlandmosaik-aus-sentinel-2-daten-wms-sentinel2-de.html has nice free (!) data as a wms service

**Comment by Lucas-Nan** [comment]
it also offers nir data which corresponds to a nicer view for vegetation specialists: https://forestryplatform.com/2022/11/09/rgb-versus-nir-what-do-these-mean/ 

![Image](https://github.com/amosproj/amos2024ss04-building-information-enhancer/assets/166795065/2455d66a-9c66-4347-b2a2-ccd76e813bd8)

**Comment by Lucas-Nan** [comment]
Oh and this whole thing means consuming the very first wms endpoint for the map

**Comment by Lucas-Nan** [comment]
so current state is a stitched version with bavaria dp40, rest of germany satellite, rest of the world in reduced intensity map (like openstreetmap)
